go/cmd/dolt/commands/commit.go

```diff
@@ -93,7 +97,7 @@ func (cmd CommitCmd) Exec(ctx context.Context, commandStr string, args []string,
 	if authorStr, ok := apr.GetValue(cli.AuthorParam); ok {
 		name, email, err = cli.ParseAuthor(authorStr)
 	} else {
-		name, email, err = actions.GetNameAndEmail(dEnv.Config)
+		name, email, err = merge.GetNameAndEmail(dEnv.Config)
```

Comment:
I don't think getting the name and the email from the config has anything to do with merging.